toolkit/partner-chains-cli/src/register/register1.rs

```diff
@@ -396,8 +396,8 @@ mod tests {
 		assert!(result.is_err());
 	}
 
-	const CHAIN_CONFIG_PATH: &str = "partner-chains-cli-chain-config.json";
-	const RESOURCE_CONFIG_PATH: &str = "partner-chains-cli-resources-config.json";
+	const CHAIN_CONFIG_PATH: &str = "pc-chain-config.json";
```

Comment:
Defined again. I understand that tests maybe would like to add extra check if production code uses expected name, but it's funny that we still use the literal on line 234.